common/religion/doctrines/00_core_tenets.txt

```diff
@@ -1650,7 +1650,7 @@
 		is_shown = {
 
 			# Warcraft
-			evil_public_religion_trigger = no	#They have it by default
+			#evil_public_religion_trigger = no	#They have it by default
```

Comment:
You fixed it wrong. They ALREADY have it from their special doctrine. Thus they shouldn't be able to pick and their religions shouldn't have it.

Reply:
I've re-disabled the tenets.
Cannibalism isn't included in the special doctrine. Should I enable that tenet for evil religions to pick or should I add it to the special doctrine? Or is it ok that they can't pick the tenet when making heresies, and the issue is instead that G'huuni should be given a different tenet?

Reply:
Nevemind, I think I figured it out. So, there's `doctrine_end_justifies_means` I talked about that all evil religions have, but Ritual Cannibalism isn't included there. Thus we should allow picking Ritual Cannibalism and that's all.